controllers/api/board-routes.js

```diff
@@ -5,21 +5,21 @@ const sequelize = require('../../config/connection');
 const withAuth = require('../../utils/auth');
 
 
-// GET /api/boards
+// GET /boards
 router.get('/', (req, res) => {
   Board.findAll({
     order: [['created_at', 'DESC']],
     attributes: [
       'id',
       'title',
       'description',
-      // 'board_content',
+      'board_content',
       'created_at',
     ],
     include: [
       {
         model: User,
-        attributes: ['username']

```

Comment:
perhaps this change is why the username isn't pulling/displaying with the boards?

Reply:
I don't think I changed that. I had changed it to id for a while but I am pretty sure that is how I found it.

Reply:
This did lead me to another place where I changed the attribute to id through, in home-routes and I that fixes the bug you are referring to. Good catch. It was like a behind the back catch :P